api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -418,6 +419,13 @@ public static PersonService getPersonService() {
 		return getServiceContext().getPersonService();
 	}
 
+	/**
```

Comment:
Can you also add the @since annotation to this?

Reply:
Okay. I am on that